app/models/configuration_project.rb

```diff
@@ -0,0 +1,2 @@
+class ConfigurationProject < ApplicationRecord
```

Comment:
Renamed! 

(@Fryguy, these comments are not lost, I can still see them -- this is one of them)